autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -835,19 +850,17 @@ def file_output(
 
         # Abort if we don't want to output anything.
         if hasattr(self, 'disable_file_output'):
```

Comment:
Define self.disable_file_output rather than just checking, because we somehow do not check `hasattr` below.

```
self.disable_file_output = [] if not hasattr(self, 'disable_file_output') else disable_file_output
```

Reply:
actually, we are assured that it is a list and we will always have the attribute `disable_file_output` as it is initialised in `__init__`. So, we can remove this `hasattr` check altogether